cernopendata/modules/records/serializers/schemas/schemaorg_schemas.py

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of CERN Open Data Portal.
+# Copyright (C) 2018 CERN.
+#
+# CERN Open Data Portal is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# CERN Open Data Portal is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with CERN Open Data Portal; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+"""CERN Opendata schema.org marshmallow schema."""
+
+from __future__ import absolute_import, print_function
+
+from marshmallow import Schema, fields
+
+from marshmallow.decorators import post_dump
+
+
+class RecordSchemaorgSchema(Schema):
+    """schema.org JSON-LD schema for CERN Opendata Record."""
+
+    # JSON-LD minimum requirements
+    context_ = fields.Field(default='https://schema.org/', dump_to='@context')
+    id_ = fields.Method('get_identifier', dump_to='@id')
+
+    # Minimum functional
+    type_ = fields.Method('get_type', dump_to='@type')
+    name = fields.Str(attribute='title', dump_to='name')
+    description = fields.Method('get_description', dump_to='description')
+    identifier = fields.Method('get_identifier', dump_to='identifier')
+    url = fields.Method('get_url', dump_to='url')
+    creator = fields.Method('get_creator', dump_to='creator')
+    date_created = fields.Str(attribute='date_published',
```

Comment:
Both `date_created` and `date_published` are taken from the same field.  We should probably use:

```python
date_created = fields.Str(attribute='date_created',
                         dump_to='dateCreated')
```

which will give the data-taking year there...

Reply:
... otherwise things work well locally 👍

Reply:
Actually, this was intentional, but I cannot anymore remember why :)
I changed to `date_created`.